### PR TITLE
Tag - fix treating complex tag as simple PhpDoc tag

### LINKED ISSUE #1147: PhpdocScalarFixer.php: Notice: Undefined offset: 1
Using last PHP-CS-Fixer on a symfony project with php 5.6: 

Notice: Undefined offset: 1 in phar://C:/[...]/php-cs-fixer.phar/Symfony/CS/Fixer/Symfony/PhpdocScalarFixer.php on line 92
PHP Notice:  Undefined offset: 1 in phar://C:/[...]/php-cs-fixer.phar/Symfony/CS/Fixer/Symfony/PhpdocScalarFixer.php on line 92

### PR DESCRIPTION
**Pull request by keradus**
Fixes #1147 and #1155

### CODE REVIEW

File: Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php [hunk 1]
```diff
@@ -195,6 +195,18 @@ public function testComplexMix()
         $this->makeTest($expected, $input);
     }
 
+    public function testComplexTag()
+    {
+        $expected = <<<'EOF'
+<?php
+    /**
+     * @Type("string")
```
Comment (GrahamCampbell) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: maybe put "boolean" in there to see there's no change

Comment (GrahamCampbell) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: Or better, just have two test cases. :P

Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: done !

**Comment by GrahamCampbell** [comment]
:+1: :ship: :rocket:

File: Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php (hunk 1 above)
Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: one would be enough

File: Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php [hunk 2]
```diff
@@ -195,6 +195,18 @@ public function testComplexMix()
         $this->makeTest($expected, $input);
     }
 
+    public function testComplexTag()
```
Comment (GrahamCampbell) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: Maybe `testComplexTagIsNotModified` would be clearer? Only a minor thing. :)

Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php]: made it ` testDoNotModifyComplexTag` to be like other methods in this class ;)

**Comment by keradus** [comment]
@fabpot please take a look at fabbot's status. It fails and tries to fix type notation inside strings like `integer` into `int`.
Our PHP CS Fixer already done that by `PhpdocScalarFixer` that does not modify strings.
Please, could you consider removing that part of fabbot in favor of that fixer? One bug less and it should be little faster (not doing same thing twice)

**Comment by fabpot** [comment]
Done... and your PR is now green :)

**Comment by keradus** [comment]
Great to hear that, thank you @fabpot !

**Pull request closed without merging**